common/workunit/workunit.cpp

```diff
@@ -14467,4 +14467,55 @@ std::pair<std::string, unsigned> getExternalService(const char *serviceName)
     return servicePair;
 }
 
+std::vector<std::vector<std::string>> getPodNodes(const char *selector)
```

Comment:
std:vector<std::pair<std:string, std:string>> seems more natural.  And using a typedef for the element.
Slightly concerned about the efficiency or reallocation (and using pair is possibly worse))

Reply:
Could do with a comment describing what the function returns.